app/lib/manage-cdo-helpers.js

```diff
@@ -0,0 +1,10 @@
+const { statuses } = require('../constants/cdo/status')
```

Comment:
These methods would have sat nicely in route-helpers but you can keep them here

Reply:
good point, happy to move later